cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CertUtils.java

```diff
@@ -226,6 +226,20 @@ public static void createTrustedCertificatesVolumes(List<Volume> volumeList, Lis
         }
     }
 
+    /**
+     * Creates the Secret Volumes for OAuth trusted TLS certificates
+     *
+     * @param volumeList            List where the volumes will be added
+     * @param isOpenShift           Indicates whether we run on OpenShift or not
+     * @param secretName            Name of the internal secret for storing trusted certificates for OAuth server
+     */
+    public static void createOauthTrustedCertificatesVolumes(List<Volume> volumeList, boolean isOpenShift, String secretName) {
```

Comment:
Why do we need this method? It doesn't seem to do anything specific to oauthTrustedCertificates, can't we call `addNewVolume(List<Volume> volumeList, String volumeNamePrefix, String secretName, boolean isOpenShift)` passing an empty string for `volumeNamePrefix`?